service-commands/src/commands/service-commands.json

```diff
@@ -102,6 +102,15 @@
     ],
     "register": [
       "cd libs/; node initScripts/local.js register-discprov-1"
+    ],
+    "restart": [
+      "cd discovery-provider; docker-compose -f compose/docker-compose.base.override.yml -f compose/docker-compose.dev.override.yml -f compose/docker-compose.ipfs.override.yml down",
+      "docker volume prune -f",
+      "cd libs/initScripts; node configureLocalDiscProv.js",
```

Comment:
are you sure this has to be reconfigured?

Reply:
eh not really, just did quickest path to getting it working